src/pages/myNotes/myNotesContainer.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import MyNotes from "./myNotes";
+import MyNoteDesctiption from "./myNotesDesctiption";
+import Notes from "./constants/notes";
+import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import PropTypes from "prop-types";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: "center",
+    color: theme.palette.text.secondary,
+  },
+}));
+
+function MyNotesContainer() {
+  const classes = useStyles();
+
+  const [notes] = useState(Notes);
+  const [activeNoteId, setActiveNoteId] = useState(-1);
+  const getActiveNote = () => {
+    return notes.find(({ id }) => id === activeNoteId);
+  };
+  return (
```

Comment:
Container should contain only logic. Can you please move all the tags inside the component?
Here we should leave only 
`return ( <MyNotes ... />)`